tensorflow/tensorboard/components/tf-event-dashboard/tf-event-dashboard.html

```diff
@@ -110,6 +110,12 @@
                       icon="fullscreen"
                       on-tap="toggleSelected"
                     ></paper-icon-button>
+                      <paper-icon-button
+                              class="log-option-button"
+                              icon="line_weight"
```

Comment:
this should be "line-weight"; that's why it isn't displaying.

I think the icon should reflect the current state of the UI, not the state that can be toggled to. So if we go with this UI approach, it should start with "re-order" (because it is a linear scale) and then swap to "line-weight" when it is a log scale.